app/src/features/apiClient/screens/apiClient/components/sidebar/components/collectionsList/collectionRow/CollectionRow.tsx

```diff
@@ -63,6 +66,17 @@ export const CollectionRow: React.FC<Props> = ({ record, onNewClick, onExportCli
     [setIsDeleteModalOpen, updateRecordToBeDeleted]
   );
 
+  const collapseChangeHandler = useCallback((keys: string[]) => {
+    if (isEmpty(keys)) {
+      collapsedKeys.push(record.id);
+    } else {
+      const activeKeyIndex = collapsedKeys.indexOf(record.id);
+      collapsedKeys.splice(activeKeyIndex, 1);
+    }
+    setActiveKey(keys[0]);
+    isEmpty(collapsedKeys) ? window.sessionStorage.removeItem("keys") : setToSessionStorage("keys", collapsedKeys);
```

Comment:
Can we rename the key to either `collection_keys` or `collapsed_collection_keys`?